v2/cli/term/event_page.go

```diff
@@ -105,6 +122,34 @@ func (e *eventPage) fillUsage() {
 	e.usage.SetText("[yellow](F5) [white]Reload    [yellow](<-/Del) [white]Back    [yellow](ESC) [white]Home    [yellow](Q) [white]Quit")
 }
 
+func (e *eventPage) fillEventInformation(event core.Event) {
+	e.eventInfo.Clear()
+	e.eventInfo.SetText(
+		fmt.Sprintf(
+			"[yellow]Event: [white]%s\n"+
+				"[yellow]Source: [white]%s\n"+
+				"[yellow]Type: [white]%s\n"+
+				"[yellow]Time Created: [white]%s",
+			event.ID,
+			event.Source,
+			event.Type,
+			event.Created.Local().Format("2006-01-02 15:04:05"),
```

Comment:
Looks like there are a few opportunities to use the helper you just added in #4

Reply:
@krancour there are some merge conflicts in the event page file. Should I remove all notions of a jobs pipeline or jobs list, or should you handle the conflicts?

Reply:
Yup. I removed the "graphical" representation of the pipeline because, although it looks fancy, it adds no value. A real timeline would depict _when_ each job starts and stops, but that was only showing how long each job ran-- and there's already a duration column. I also removed it to make room for these event details.